protoc-gen-gqlgencfg/main.go

```diff
@@ -35,8 +47,8 @@ func main() {
 	fileLen := len(gen.Response.GetFile())
 	for i := 0; i < fileLen; i++ {
 		gen.Response.File[i].Name = proto.String(strings.Replace(gen.Response.File[i].GetName(), ".pb.go", ".gqlgen.pb.yml", -1))
-		_, file := path.Split(strings.Replace(gen.Response.File[i].GetName(), ".gqlgen.pb.yml", ".pb.graphqls", -1))
-		p.config[i].SchemaFilename = []string{path.Join(".", file)}
+		file := strings.Replace(gen.Response.File[i].GetName(), ".gqlgen.pb.yml", ".pb.graphqls", -1)
```

Comment:
I don't think we need to delete this. The idea was that the path to gql file has to be in the same directory as other generatedd files